resources/deployment.yaml

```diff
@@ -47,6 +50,9 @@ spec:
         imagePullPolicy: IfNotPresent
         args:
         - "--v=5"
+        envFrom:
+        - secretRef:
+            name: objectstorage-provisioner
```

Comment:
I'm still trying to figure out how this secret is getting propagated into the applications. Does this env need to be referenced in the `args` like `- "--driver-prefix=$(DRIVER_PREFIX)`?